core/shared/src/main/scala/spire/std/float.scala

```diff
@@ -6,7 +6,7 @@ import spire.math.Rational
 
 import java.lang.Math
 
-trait FloatIsField extends Field[Float] {
+trait FloatIsField extends Field.WithDefaultGCD[Float] {
```

Comment:
Yeah - I think 2) may be an OK way to go. Specifically, I think the error on the final GCD is actually relative to the result +/- eps or something very near. If we merge this with the code commented out, I can make a PR to add a test that re-enables the old behaviour, but adds a test that it's approximately equal to the `Rational` case w/ 2^n denom.